style.css

```diff
@@ -1,825 +1,810 @@
-  padding: 10px 23px;
-  margin-right: 52px;
+.second__block-color-pink {
+	background-color: #f29f9f;
+	margin-left: 9.14px;
 }
 
-.collection__other-text {
-  display: inline-block;
-  text-decoration: none;
-  color: black;
+.second__block-color-blue {
+	background-color: #a8bff9;
+  margin-left: 9.14px;
+}
+
+.second__block-color-green {
+  background-color: #90bd8f;
+  margin-left: 9.14px;
+}
+
+.second__block-coust {
+	font-size: 20px;
+	font-weight: 900;
+	margin: 0;
+}
+
+.second__block {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-bottom: 42px;
+}
+
+.second__add {
+  font-weight: 500;
   font-size: 13px;
-  letter-spacing: 0;
-  margin-right: 14px;
+  text-decoration: none;
+  color: #000;
+  padding: 10px 48px;
+  border: 1px solid #000;
+  margin-left: 31px;
 }
 
-.collection__scroll {
-  position: absolute;
-  width: 27px;
-  height: 46px;
-  left: 49%;
-  top: 815px;
-  border: 2px solid #000;
-  box-sizing: border-box;
-  border-radius: 15px;
+.third {
+  min-width: 320px;
+  height: 597px;
+  background-color: #ebe3d8;
+  position: relative;
 }
 
-.collection__scroll-elem {
-  position: absolute;
-  width: 2.45px;
-  height: 8.21px;
-  left: 50%;
-  top: 826.5px;
+.third__line {
+  width: 41px;
+  height: 597px;
   background: #000;
-  border-radius: 19px;
+  margin-left: 100px;
 }
 
-.collection__next-text:hover,
-.collection__other-text:hover {
-  opacity: .5;
-  transition: 1s ease-out .2s;
+.third__img {
+	position: absolute;
+  min-width: 320px;
+  right: 12px;
+  left: 12px;
+  top: 34px;
 }
 
-.vectormob {
-  display: none;
+.third__img>div {
+  background: url(img/third-img.png) no-repeat;
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  height: 262px;
 }
 
-@media (max-width: 1235px) {
-  .first {
-    max-width: 1235px;
-    width: 100%;
-    display: grid;
-    grid-template-columns: 35.2% 64.8%;
-    grid-template-rows: 719px;
-  }
+.third__block {
+  width: 305px;
+  height: 195px;
+  background: #fae8e4;
+  top: 274px;
+  position: absolute;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+}
 
-  .header__logo {
-    font-size: 24px;
-    letter-spacing: 2px;
-    margin: 5px 41px;
-  }
+.third__container {
+	width: 256px;
+	height: 115px;
+	margin: 0 auto;
+}
 
-  .header__menu {
-    height: 74px;
-    width: 74px;
-  }
+.third__title {
+  font-size: 20px;
+  line-height: 1;
+  margin-bottom: 0;
+  letter-spacing: .4px;
+  margin-top: 24px;
+}
 
-  .header__menu-item {
-    height: 3px;
-    width: 28.7px;
-    margin-bottom: 4px;
-  }
+.third__subtitle {
+  font-size: 10px;
+  line-height: 1;
+  letter-spacing: 0.07em;
+  opacity: .5;
+  margin-top: 7px;
+  margin-bottom: 11px;
+}
 
-  .header__language {
-    left: 29.5%;
-    top: 34px;
-    font-size: 16px;
-  }
+.third__text {
+	font-size: 9px;
+	line-height: 1.44;
+	opacity: .5;
+}
 
-  .header__vector {
-    left: 32.5%;
-    top: 42px;
-  }
+.third__add {
+	font-weight: 500;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)